app/modules/power/scripts/power

```diff
@@ -0,0 +1,380 @@
+#! /usr/bin/python
+# power script by tuxudo for MunkiReport
+
+import subprocess
+import plistlib
+import os
+import sys
+import string
+import re
+
+def get_battery_profiler():
+    '''Uses system profiler to get power info for this machine.'''
+    cmd = ['/usr/sbin/system_profiler', 'SPPowerDataType', '-xml']
+    proc = subprocess.Popen(cmd, shell=False, bufsize=-1,
```

Comment:
depending on the information you want to get from subprocess, it can be cleaner to use the `check_output` method which only cares about the output of the command you are running.